src/Json.Schema.Validation/RuleResources.resx

```diff
@@ -200,117 +200,117 @@
   </data>
   <data name="RuleDescriptionTooFewProperties" xml:space="preserve">
-    <value>An object has fewer properties than permitted by the schema's "minProperties" property.</value>
+    <value>An object has fewer properties than permitted by the schema's 'minProperties' property.</value>
   </data>
   <data name="RuleDescriptionTooManyArrayItems" xml:space="preserve">
-    <value>An array has more elements than permitted by the schema's "maxItems" property.</value>
+    <value>An array has more elements than permitted by the schema's 'maxItems' property.</value>
   </data>
   <data name="RuleDescriptionTooManyProperties" xml:space="preserve">
-    <value>An object has more properties than permitted by the schema's "maxProperties" property.</value>
+    <value>An object has more properties than permitted by the schema's 'maxProperties' property.</value>
   </data>
   <data name="RuleDescriptionValidatesAgainstNotSchema" xml:space="preserve">
-    <value>An instance successfully validates against the schema specified by the schema's "not" property.</value>
+    <value>An instance successfully validates against the schema specified by the schema's 'not' property.</value>
   </data>
   <data name="RuleDescriptionValueTooLarge" xml:space="preserve">
-    <value>A numeric value is greater than the maximum value permitted by the schema's "maximum" property.</value>
+    <value>A numeric value is greater than the maximum value permitted by the schema's 'maximum' property.</value>
   </data>
   <data name="RuleDescriptionValueTooLargeExclusive" xml:space="preserve">
-    <value>A numeric value is greater than or equal to the exclusive maximum value permitted by the schema's "maximum" and "exclusiveMaximum" properties.</value>
+    <value>A numeric value is greater than or equal to the exclusive maximum value permitted by the schema's 'maximum' and 'exclusiveMaximum' properties.</value>
   </data>
   <data name="RuleDescriptionValueTooSmall" xml:space="preserve">
-    <value>A numeric value is less than the minimum value permitted by the schema's "minimum" property.</value>
+    <value>A numeric value is less than the minimum value permitted by the schema's 'minimum' property.</value>
   </data>
   <data name="RuleDescriptionValueTooSmallExclusive" xml:space="preserve">
-    <value>A numeric value is less than or equal to the exclusive minimum value permitted by the schema's "minimum" and "exclusiveMinimum" properties.</value>
+    <value>A numeric value is less than or equal to the exclusive minimum value permitted by the schema's 'minimum' and 'exclusiveMinimum' properties.</value>
   </data>
   <data name="RuleDescriptionWrongType" xml:space="preserve">
-    <value>An instance has a type that is not permitted by the schema's "type" property.</value>
+    <value>An instance has a type that is not permitted by the schema's 'type' property.</value>
```

Comment:
>instance [](start = 14, length = 8)

what's the bad type?

Reply:
This is all great. I didn't want to make more investment in these messages than you'd approve, but since we have the hood up and we both have some energy around this, I'll take a couple of hours and do some deeper cleaning (including functional test assets as you suggested in email).

I'll merge this to capture what we have, then go on and do those things.

Reply:
Actually @michaelcfanning, what you're looking at here is the rule description. The result message does indeed mention the type, as in this example from the comprehensive end-to-end suite I'm working on:
```
c:\dev\jschema\src\Json.Schema.Validation.UnitTests\TestData\ComprehensiveInstanceFile.json(22,30):
error JSON1001: testSuites.negative.JSON1001_WrongType.integerProperty:
The schema requires one of the types (Integer), but a token of type 'String' was found.
```